src/components/eachBook.js

```diff
@@ -1,31 +1,74 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// import './styles/Book.css';
 import { useDispatch } from 'react-redux';
-import { deleteBookFetch} from '../redux/Books/Books';
+// import { Progress } from 'antd';
```

Comment:
OPTIONAL
_I make it optional but this is a general requirement for every project._
- [ ] To make clean code kindly delete commented code.